Extension/SourceEditorCommand.swift

```diff
@@ -43,7 +40,7 @@ class SourceEditorCommand: NSObject, XCSourceEditorCommand {
             for i in 0 ..< selections.count {
                 if let selection = selections[i] as? XCSourceTextRange, selection.start != selection.end {
                     hasSelection = true
-                    for j in selection.start.line ... selection.end.line {
+                    for j in selection.start.line...selection.end.line {
```

Comment:
This is how [Apple does it](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/BasicOperators.html#//apple_ref/doc/uid/TP40014097-CH6-ID73).